todo_app/data/session_items.py

```diff
@@ -1,78 +1,67 @@
+import requests
 from flask import session
```

Comment:
Unused import.

Also you should have noticed the README.md contains this:
> There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.

But the SECRET_KEY is now irrelevant since you do not use the session cookie. flask_config.py and any reference to SECRET_KEY can be deleted.

Ideally this file would also be renamed since there is no "session".